atlassian_jwt_auth/frameworks/common/exceptions.py

```diff
@@ -0,0 +1,5 @@
+from atlassian_jwt_auth.exceptions import ASAPAuthenticationException
+
+
+class NoTokenProvidedError(ASAPAuthenticationException):
```

Comment:
Well arguably our existing exceptions inherit from `ValueError` (https://github.com/atlassian/asap-authentication-python/blob/master/atlassian_jwt_auth/exceptions.py)

Reply:
Sure. I'm not strongly opposed to it. I only put it here because it is thrown and caught only within this package.